### ISSUE TITLE
The content submission is in progress but nothing happen - Exception: Invalid content.json file

### ISSUE THREAD

**Issue by slrslr** [HelloZeroNet/ZeroNet#2675]
### Step 1: Please describe your environment

  * ZeroNet version: 0.7.2 r4548, Python 3.7.3
  * Operating system: Linux Arch
  * Web browser: FF
  * Tor status: available
  * Opened port: yes
  * Special configuration: multiuser

#### Steps to reproduce:

i have tried to like someone's topic from within index page of the ZeroTalk. It was only animating the vote button and nothing else.
The debug.log though shows something relevant:
> Site:1TaLkF..jipT WebSocket handleRequest error: Exception: Invalid content.json file: data/users/myaddress/data.json in UiWebsocket.py line 80 > 236 > MergerSite/MergerSitePlugin.py line 212 > 206 > UiWebsocket.py line 473 > MergerSite/MergerSitePlugin.py line 209 > 206 > OptionalManager/UiWebsocketPlugin.py line 41 > UiWebsocket.py line 415 
>  {"cmd":"sitePublish","params":{"inner_path":"data/users/myaddress/data.json"},"wrapper_nonce":"somestringhere","id":11}
> 

So maybe ZeroNet can either fix the file on background or print some error on page so one knows how to fix this issue...

I have checked that data.json using online validator and it says it is valid and content.json seems OK too.

**Comment by slrslr** [comment]
I have moved my profile folder out of ZeroTalk site (zeronet/data/1TaLkFrMwvbNsooF4ioKAY9EuxTBTjipT/data/users/myfolder) and then tried to Recheck files from within the three dots menu near the site name on ZeroHello. I then tried to click Rebuild database from within onsite (0) menu, Reload Database, Update button. (i do nto know which one was necessary, it is confusing for regular user). And it apparently **it downloaded my valid json files from other peers and liking someone topic now works**.

when comparing current files and ones from non-working profile, content.json has different timestap and different second string in the signers line (ending with =). data.json had some differences too, shown [here](https://bin.disroot.org/?c06e6943fc5dff93#BAiEANSdzD3ZXKPHxv38wkwtzetQQP4b81txReS3azrn) using linux diff command (pass: private).

Though i think this issue should still be treated somehow because not everyone knows he/she should do what i did.

Thank you

**Comment by HelloZeroNet** [comment]
This error got displayed when for some reason it can't get file info for `data/user/../data.json` file. (probably not imported correctly)

Based on the diff your one file has and extra topic in it, does the voting still works if you add that topic to the downloaded and fixed file?

**Comment by slrslr** [comment]
Yes, i have copied that topic text and title from the problematic data.json and via ZeroTalk UI added the topic, maybe with some small modification and that worked.. Though i was unable to find someone else's topic to which i also commented before so i have not retried that comment (which was also difference between problematic and valid/downloaded data.json). I guess this info. does not help you much..
If important, i can try different thing like using "problematic" data.json and try to sign it, but i am unsure which are all steps needed to do or if this would help u anyhow.

**Comment by slrslr** [comment]
update: i tried to replace my current/valid data.json by the one that i believe was there during the error described (i see it contains the appropriate contents seen in the diff) yes and it works, signing works, liking works. So it does not seem like problem of the data.json content to me. When i replaced both content.json and data.json by ones i believe was there during error, and like someone else topic, it worked now, no error.
regarding content.json, i mentioned it had some difference on signers line. Though may be unrelated.
